.github/workflows/release.yaml

```diff
@@ -29,15 +29,15 @@ jobs:
           # Patch the version number the source to be consistent with the tag
           sed -i.bak \
             "s/NUMBER *=.*/NUMBER = \"${VERSION}\"/" \
-            cbmc_viewer/version.py
+            src/cbmc_viewer/version.py
 
           # Patch the version number pip setup to be consistent with the tag
           sed -i.bak \
-            "s/version *=.*/version = \"${VERSION}\",/" \
-            setup.py
+            "s/version *=.*/version = ${VERSION}/" \
+            setup.cfg
 
           # Create the package
-          python3 -m pip install --upgrade setuptools wheel
+          python3 -m pip install --upgrade build setuptools wheel
```

Comment:
Can we add pip to the list of packages to upgrade to get rid of the "old pip" error messages.

Reply:
Resolved